pkg/genericcomp/genericcomp_test.go

```diff
@@ -229,6 +230,13 @@ func TestGetSecrets(t *testing.T) {
 	// Make sure fluentd is enabled so additional secrets are created.
 	fluentdEnabled := true
 	generic.FluentdEnabled = &fluentdEnabled
+	generic.Deployment.Containers[0].Ports = []corev1.ContainerPort{
```

Comment:
Where are you validating the ports?

Reply:
genericcomp.go

`// Get the targetPort value
func getTargetPort(generic *v1beta1v8o.VerrazzanoGenericComponent) int32 {

	// Default target port value is 8080
	var targetPort int32 = 8080
	if generic.Deployment.Containers[0].Ports[0].ContainerPort != 0 {
		targetPort = generic.Deployment.Containers[0].Ports[0].ContainerPort
	}

	return targetPort
}
`